.github/actions/meson_install/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Setup python meson ninja.'
```

Comment:
Should these be capitalized? Also, the quotes are inconsistent. Other places in the file use `'`, `"`, or no quotes.